### ISSUE TITLE
Include new batch systems

### ISSUE THREAD

**Issue by nils-braun** [nils-braun/b2luigi#2]
In the moment, there is only a reference implementation for LSF, although it should be very easy to include more batch systems.
Some examples include:
* htcondor
* grid (e.g. dirac)
* Amazon Batch
* Amazon Lambda
* Google Cloud
* Azure

It would also be very nice to have a batch system discovery at some point.

**Comment by nils-braun** [comment]
@welschma has implemented HTCondor in #19.
After #20 inclusion should be simplified.
